src/data.js

```diff
@@ -16,47 +16,115 @@ export const quizData = {
   // the questions in the quiz
   questions: [
     {
-      text: 'What are the different ways to declare a JS variable?',
+      text: 'What is the default value of the position property??',
       answers: {
-        a: 'constant, let, variable',
-        b: 'var, const, let, function',
-        c: 'var, let, const',
+        a: 'fixed',
+        b: 'relative',
+        c: 'absolute',
+        d: 'static',
       },
       correct: 'c',
```

Comment:
Maybe we can rename this property as `correctAnswer`. Otherwise when I read it from parent property, it sounds like question(s) -> correct, so correct question 😀

Naming is one of the most struggled and important part of programming. We should do our best to keep everything as explicit as possible to make it clear to other people who would read that code.
```suggestion
      correctAnswer: 'c',
```